core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -108,6 +120,81 @@ oppia.controller('CollectionPlayer', [
         explorationId).getExplorationSummaryObject();
     };
 
+    $scope.generatePathParameters = function() {
+      var pathSvgParameters = 'M250 80  C 500 100, 500 280, 250 300';
+      var collectionNodeCount = $scope.collection.getCollectionNodeCount();
+      var sParameterExtension = '';
+      $scope.svgHeight = 220;
```

Comment:
Where does 220 come from? Is it an arbitrary constant? If it is a constant, give it a name explaining what it means and declare it outside this function, then do $scope.svgHeight = $scope.WHATEVER_THE_NAME_IS here.